cmake/modules/SetRPATH.cmake

```diff
@@ -0,0 +1,15 @@
+set(CMAKE_SKIP_BUILD_RPATH FALSE)
+set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
```

Comment:
Both of these variables should already be set to FALSE by default. I think they are only mentioned in the CMake Wiki for clarity.

Reply:
Well, *exactly*. They are mentioned for clarity.

* Defaults are great, until they're not; convention is a terrible foundation.

* As it turns out, `cmake `has a lot of knobs related to `rpath`; one might as well set the knobs explicitly, and in one place, so as to help anybody in the future who has some *unforeseen* need.

Nevertheless:

* The reason it made sense to me to remove the explicit setting of the *target* property `MACOSX_RPATH` is that such an option never made much sense, anyway.

* If you still don&rsquo;t agree with setting some of these variables explicitly, then perhaps they can be simply commented out. Would that be acceptable?

Reply:
I think it's misleading to set variables to their default values.

Reply:
Of course, it's not misleading *at all*. Quite the opposite.

However, I'll choose to interpret your reply as "Look, just remove them, OK?"